user-service/src/main/java/uk/gov/gchq/palisade/service/user/request/GetUserRequest.java

```diff
@@ -32,9 +33,9 @@
 
 /**
  * A {@code GetUserRequest} is a {@link Request} that is passed to the user-service
- * to get an existing {@link uk.gov.gchq.palisade.User}.
```

Comment:
I prefer this as it was before e.g full qualified class in the javadoc rather than importing it when it is only needed in the javadoc